<launch>

  <node 
    pkg="localize" 
    name="location_sender" 
    type="location.py">

    <param name = "frequency" value= "50"/>
    <param name = "activate_DynamicModel" value= "1"/>
    <param name = "activate_LaneMeas" value= "0"/>

    <param name = "GPS_accept_interval" value= "1"/>
    <param name = "GPS_deny_interval" value= "0"/>
    <param name = "activate_GPS_reset" value= "0"/>

    <param name = "num_particles" value= "100"/>

    <param name = "gps_V_lateral_bias" value= "0.02"/>

    <param name = "straight_position_noise" value= "0.01"/>
    <param name = "turn_position_noise" value= "0.0005"/>
    <param name = "lateral_velocity_noise" value= "0.1"/>
    <param name = "turn_velocity_noise" value= "0.0001"/>

    <param name = "gps_X_noise" value= "0.03"/>
    <param name = "gps_Y_noise" value= "0.03"/>
    <param name = "gps_Psi_noise" value= "0.002"/>
    <param name = "gps_V_lateral_noise" value= "0.002"/>
    <param name = "dot_Psi_noise" value= "0.005"/>
    <param name = "a_r0_noise" value= "0.005"/>
    <param name = "a_l0_noise" value= "0.005"/>
    <param name = "a_r1_noise" value= "0.0005"/>
    <param name = "a_l1_noise" value= "0.0005"/>

    <param name = "mass" value= "1840.0"/>
    <param name = "Izz" value= "3477.0"/>
    <param name = "lf" value= "1.105"/>
    <param name = "lr" value= "1.738"/>

    <param name = "B_f" value= "0.1447"/>
    <param name = "B_r" value= "0.1678"/>
    <param name = "C_f" value= "0.1463"/>
    <param name = "C_r" value= "0.1709"/>
    <param name = "D_f" value= "1922700."/>
    <param name = "D_r" value= "2249000."/>
    <param name = "E_f" value= "-0.3782"/>
    <param name = "E_r" value= "-0.4420"/>

    <param name = "Psi_rel_threshold" value= "0.000001"/>
    <param name = "Lane_threshold" value= "5"/>
    <param name = "DynamicModel_threshold" value= "5"/>

    <param name = "frac_through" value= "0.1"/>

  </node>

  <!--
  <node 
    pkg="localize" 
    name="can_sender" 
    type="can_sender.py">
  </node>

  -->

</launch>
